Blackjack.Tests/ControllerShould.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Moq;
+using Xunit;
+
+namespace Blackjack.Tests
+{
+    public class ControllerShould
+    {
+        [Fact]
+        public void WhenInitialized_ReturnsADeck()
+        {
+            var deck = new Deck(new List<Card>());
+            var controller = new Controller(deck);
+            var numberOfCardsInSuit = controller.Deck.Cards.Count(x => x.Suit == CardSuit.Clubs);
+            var numberOfCardsOfRank = controller.Deck.Cards.Count(x => x.Rank == CardRank.Ace);
+
+            Assert.Equal(52, controller.Deck.Cards.Count);
+            Assert.Equal(13, numberOfCardsInSuit);
+            Assert.Equal(4, numberOfCardsOfRank);
+        }
+
+        [Fact]
+        public void WhenShuffled_DeckHasSameCards() // TODO: how to call through Controller.Play()

```

Comment:
hint: You can utilise Moq to test behaviour here, e.g to check if a function you expected to be called was indeed called.